### PR TITLE
ui: Show default nic/network in infocard

### PR DESCRIPTION
**Pull request by davidjumani**
### Description

Shows default next to the default network / nic

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Deploy VM :

Screenshot from 2021-06-07 14-48-46

#### VM Instance :

Screenshot from 2021-06-07 14-48-24

### CODE REVIEW

File: ui/src/components/view/InfoCard.vue [hunk 1]
```diff
@@ -281,6 +281,9 @@
                 style="margin-left: -24px; margin-top: 5px;">
                 <a-icon type="api" />eth{{ index }} {{ eth.ipaddress }}
                 <router-link v-if="eth.networkname && eth.networkid" :to="{ path: '/guestnetwork/' + eth.networkid }">({{ eth.networkname }})</router-link>
+                <span v-if="eth.isdefault">
```
Comment (rhtyd) [inline ui/src/components/view/InfoCard.vue:284]: @davidjumani the default I think is usually the first; more clearly seen from the nics tab.

Comment (rhtyd) [reply ui/src/components/view/InfoCard.vue:284]: Instead of parenthesis can you use a tiny tag https://www.antdv.com/components/tag/

Comment (davidjumani) [reply ui/src/components/view/InfoCard.vue]: Added tags for nics but not for networks so it maintains the theme

Screenshot from 2021-06-08 10-06-42
Screenshot from 2021-06-08 10-06-53

**Comment by davidjumani** [comment]
@blueorangutan ui

**Review by utchoang — APPROVED** [review]
LGTM!

**Review by GabrielBrascher — APPROVED** [review]
Works as expected, LGTM

**Comment by DaanHoogland** [comment]
@GabrielBrascher you are the second lgtm and imply that you tested, any reason you didn't merge?

**Pull request merged**